src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.EasyRequest.cs

```diff
@@ -205,6 +205,14 @@ private void SetVerb()
                         SetCurlOption(CURLoption.CURLOPT_COPYPOSTFIELDS, string.Empty);
                     }
                 }
+                else
+                {
+                    SetCurlOption(CURLoption.CURLOPT_CUSTOMREQUEST, _requestMessage.Method.Method);
+                    if (_requestMessage.Content != null)
+                    {
+                        SetCurlOption(CURLoption.CURLOPT_UPLOAD, 1L);
```

Comment:
> The HTTP RFC doesn't explicitly ban entity-body from GET requests. But most servers don't allow it.

Thanks, David.  What does WinHttpHandler do here if content is provided with a GET request?  Sends it and let's the server respond as it will?